### PR TITLE
Equipment

### PR DESCRIPTION
**Pull request by dabick14**

### CODE REVIEW

File: api/BeforeNextUpdate.cypher [hunk 1]
```diff
@@ -1,3 +1,9 @@
+//Equipment Campaign
+ //Adding constraint for equipment campaign
+CREATE CONSTRAINT con_equipment_record_id FOR (n:EquipmentRecord) REQUIRE n.id IS UNIQUE;
+
+
+
 //Adding CONSTRAINST
```
Comment (jaedag) [inline api/BeforeNextUpdate.cypher:7]: Typo

File: web-react-ts/src/pages/directory/quick-facts/QuickFactsQueries.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { gql } from '@apollo/client'
 
-export const FELLOWSHIP_AVG_WEEKDAY_ATTENDANCE_THIS_MONTH = gql `
```
Comment (jaedag) [inline web-react-ts/src/pages/directory/quick-facts/QuickFactsQueries.ts:3]: This change of yours is going to lead to merge conflicts

**Review by jaedag — APPROVED** [review]

File: web-react-ts/src/pages/directory/quick-facts/QuickFactsQueries.ts (hunk 2 above)
Comment (dabick14) [reply web-react-ts/src/pages/directory/quick-facts/QuickFactsQueries.ts:3]: I have no idea what I was doing there. Will check please.

File: api/BeforeNextUpdate.cypher (hunk 1 above)
Comment (dabick14) [reply api/BeforeNextUpdate.cypher:7]: I'm not able to see the typo.

**Pull request merged**